static/app/__mocks__/react-date-range.tsx

```diff
@@ -1,5 +1,5 @@
 import type {CalendarProps, DateRangeProps, Range, RangeKeyDict} from 'react-date-range';
-import format from 'date-fns/format';
+import moment from 'moment-timezone';
```

Comment:
we somehow bring in `date-fns`, and while I think it’s the better lib than `moment`, all of sentry uses `moment` right now, so I didn’t just want to add `date-fns` as a dependency to `package.json` and cause confusion.

Reply:
react date range uses date-fns under the hood. some weird version of it. might have been what happened here